src/Symfony/Component/HttpFoundation/Tests/Session/Storage/Handler/NativeSessionHandlerTest.php

```diff
@@ -31,11 +31,11 @@ public function testConstruct()
         // note for PHPUnit optimisers - the use of assertTrue/False
         // here is deliberate since the tests do not require the classes to exist - drak
         if (\PHP_VERSION_ID < 50400) {
-            $this->assertFalse($handler instanceof \SessionHandler);
-            $this->assertTrue($handler instanceof NativeSessionHandler);
+            $this->assertNotInstanceOf('SessionHandler', $handler);
+            $this->assertInstanceOf('Symfony\Component\HttpFoundation\Session\Storage\Handler\NativeSessionHandler', $handler);
```

Comment:
IMO this assertion could as well just be dropped

Reply:
And just now I saw the note:
```
// note for PHPUnit optimisers - the use of assertTrue/False
// here is deliberate since the tests do not require the classes to exist - drak
```